cdap-ui/app/cdap/components/RouteToNamespace/index.js

```diff
@@ -46,23 +46,31 @@ export default class RouteToNamespace extends Component {
     if (!list || list.length === 0) { return; }
 
     /**
-     * 1. Check if localStorage has a 'DefaultNamespace' set by the user, if not,
-     * 2. Check if there is a 'default' namespace from backend, if not,
-     * 3. Take first one from the list of namespaces from backend.
+     * 1. Check if localStorage has an already selected namespace set by the user, if not,
+     * 2. Check if localStorage has a 'DefaultNamespace' set by the user, if not,
```

Comment:
Yes thats why I mentioned it doesn't make sense now but when we start to use the user store and when the user goes to a new browser or environment they will always land on their default namespace rest of the time we will check if there is a NS (last accessed namespace) in localstorage. If yes then we would go to that namespace.